custom_components/toyota/sensor.py

```diff
@@ -281,6 +289,9 @@ def native_value(self):
 class ToyotaCurrentWeekSensor(StatisticsBaseEntity):
     """Class for current week statistics sensor."""
 
+    _attr_device_class = SensorDeviceClass.DISTANCE
+    _attr_native_unit_of_measurement = LENGTH_KILOMETERS

```

Comment:
The data can also be in miles

Reply:
You are right. I removed the unit information.